test/performance/odp_ipsecfwd.c

```diff
@@ -336,39 +385,35 @@ static void print_usage(void)
 	       "  -m, --mode          Queueing mode.\n"
 	       "                          0: ordered (default)\n"
 	       "                          1: parallel\n"
-	       "  -s, --sa            SA configuration file. Individual SA configuration is\n"
-	       "                      expected to be within a single line, values whitespace\n"
-	       "                      separated:\n"
-	       "\n"
-	       "                          <line in file> Dir SPI TunSrcIPv4 TunDstIPv4"
-	       " CipherAlgoIdx CipherKey CipherKeyExtra AuthAlgIdx AuthKey AuthKeyExtra ICVLen\n"
+	       "  -C, --conf          Configuration file. 'libconfig' syntax is expected.\n"
+	       "                      SA configuration supports default fallback, i.e.\n"
+	       "                      individual SA configuration blocks may omit some\n"
+	       "                      parameters and instead set these once in default block\n"
+	       "                      which then are used to fill missing parameters. Only\n"
```

Comment:
Add "the": "The only required argument..."

Reply:
V9.